<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCSVImport</class>
 <widget class="QDialog" name="DialogCSVImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="labelFileNotFound">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Could not open the selected file.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBoxCSVSeparator">
       <item>
        <property name="text">
         <string>,</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>;</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAddSeparator">
       <property name="toolTip">
        <string>Add a custom separator.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="spinBoxTimestampColumn">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; If your dataset contains timestamps URH will calculate the sample rate from them. You can manually edit the sample rate after import in the signal details.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="specialValueText">
      <string>Not present</string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>I Data Column:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="spinBoxQDataColumn">
     <property name="specialValueText">
      <string>Not present</string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; If your dataset contains timestamps URH will calculate the sample rate from them. You can manually edit the sample rate after import in the signal details.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Timestamp Column:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Q Data Column:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidgetPreview">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Timestamp</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>I</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Q</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditFilename"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnChooseFile">
       <property name="text">
        <string>...</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spinBoxIDataColumn">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>CSV Separator:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxFilePreview">
     <property name="title">
      <string>File Content (at most 100 rows)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditFilePreview">
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>File to import:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="btnAutoDefault">
     <property name="text">
      <string>Prevent Dialog From Close with Enter</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditFilename</tabstop>
  <tabstop>btnChooseFile</tabstop>
  <tabstop>plainTextEditFilePreview</tabstop>
  <tabstop>comboBoxCSVSeparator</tabstop>
  <tabstop>btnAddSeparator</tabstop>
  <tabstop>spinBoxIDataColumn</tabstop>
  <tabstop>spinBoxQDataColumn</tabstop>
  <tabstop>spinBoxTimestampColumn</tabstop>
  <tabstop>tableWidgetPreview</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCSVImport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCSVImport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
